pkg/registry/apis/provisioning/repository/repository.go

```diff
@@ -15,6 +15,10 @@ import (
 	provisioning "github.com/grafana/grafana/pkg/apis/provisioning/v0alpha1"
 )
 
+// FIXME: the name of the mock is different because there is another generated mock for Repository
+// I don't know how it got generated.
```

Comment:
🙃 

should we fix/change in this PR, or easier in another?

Reply:
will fix separately because that mock is used in a lot of tests already :)